clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupResult.java

```diff
@@ -77,7 +74,7 @@
         this.future.whenComplete((memberErrors, throwable) -> {
             if (throwable != null) {
                 result.completeExceptionally(throwable);
-            } else if (!maybeCompleteExceptionally(memberErrors, convertToMemberIdentity(member), result)) {
+            } else if (!maybeCompleteExceptionally(memberErrors, member.getMemberIdentity(), result)) {
```

Comment:
Hmm.. When a user constructs `GroupMember`, they do not know the memberId. So this conversion here will result in an incomplete `MemberIdentity` object, so will the map lookup succeed?

Reply:
I choose to disable the member.id propagation in the admin request so that the query could succeed.